app/controllers/projects_controller.rb

```diff
@@ -318,8 +322,8 @@ def fork
       redirect_to child.urlbase
       # todo - notifications
     else
-      flash[:alert] = "Couldn't fork project, try again."
-      redirect_to @project
+      flash[:alert] = "Couldn't fork project, you already have a project with the same name."
```

Comment:
@rohitpaulk yeah it's possible but do we need to? ;)
I can't think of real case where the validations could fail except that.